GraphDB_Engine/Compute/PullFromRepo.cs

```diff
@@ -75,7 +79,7 @@ public static SparqlResultSet PullFromRepo(string serverAddress = "http://localh
             }
             catch (RdfQueryException queryEx)
             {
-                Log.RecordError($"The query was invalid. Message " + queryEx.Message, innerException: queryEx);
+                Log.RecordError($"The query was invalid. Message " + queryEx.Message); 
```

Comment:
Should be left as it was, it's for logging the actual exception, which has useful additional debug properties, other than just the exception message we would otherwise record alone. I don't see any compile error by leaving as it was, was there another reason why you changed it?

Reply:
Somehow was a problem last time, but works fine as it was now. Changed it back.